### DISCUSSION TITLE
5573-1 Kennt jemand die Party Zeit Register

### DISCUSSION THREAD

**Discussion by FWallenwein** [Tom-Bom-badil/samson_trovis_557x#54] [General (de/en)]
Hallo zusammen, 
ich bin gerade dabei, eine Fernanzeige und Steuerung mit M5Stack zu bauen. 
Ich möchte u.a. gerne 
* die Wassertemperatur anzeigen ( das klappt schon )
* über Tastendruck manuell das Wasser außerhalb der eingestellten Zeiten erwärmen können ( das fehlt noch)

Die Idee wäre, die Funktion "Partyzeit" zu nutzen. Damit könnte man einmalig einen Befehl runter schreiben, und das Wasser würde automatisch die eingestellte Zeit lang aufgeheizt. Würde ersparen, die Betriebsart umzuschalten, zu überwachen und wieder zurück zu schalten. 

Über die Funktion "Partyzeit" habe ich nirgends etwas finden können. Weiß jemand, ob die Funktion über Modbus Register zugänglich ist ? 

Danke und Gruss
Frank

**Comment by Tom-Bom-badil** [comment]
Hab ich ehrlich gesagt nie drüber nachgedacht. Die Sonder-TW-Temp scheint in 1808 zu stehen - aber das Einschalten geschieht vermutlich über die Betriebsart. Ich kenne da bisher nur 0-5 sowie 10 und 11 - den Rest hab ich nie ausprobiert (Stellung 'Party'). Einfach mal einschalten und auslesen ... 

/tom
